src/main/java/com/flansmod/client/model/ModelVehicle.java

```diff
@@ -60,12 +74,23 @@ public void render(DriveableType type)
 		renderPart(drillHeadModel);
 		for(ModelRendererTurbo[] mods : ammoModel)
 			renderPart(mods);
+		for(ModelRendererTurbo[] latm : leftAnimTrackModel)
```

Comment:
Is that what this part is for? It should be changed to render whatever frame of animation we are in (should default to 0 anyway).